_data/guides.yml

```
@@ -13,3 +13,11 @@
   links:
     - url: https://docs.google.com/document/d/1-6e7QAhklcXMxr3H05808iVopLy65atuhBNH7cvUCyY/edit
       text: "How to: Travel with the UW (intranet)"
+- title: CHTC
+  desc: Resources for using the Center for High Throughput Computing (CHTC) at UW-Madison
```

Need this additional entry to make it work:
```suggestion
  desc: Resources for using the Center for High Throughput Computing (CHTC) at UW-Madison
  links:
```